### ISSUE TITLE
Error loading type Typespec 0x1b000036 from ...

### ISSUE THREAD

**Issue by AmansRevenger** [tidusjar/PlexRequests.Net#520]
#### Plex Requests.Net Version:

1.9.x

#### Operating System:

Debian jessie on a raspberry pi

#### Mono Version:

3.10

#### Applicable Logs (from `/logs/` directory or the Admin page):

```
2016/09/01 12:21:10.483 PlexRequests.UI.Startup Fatal: System.NullReferenceException: Object reference not set to an instance of an object
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.get_ParentDefinitionMethodInfo () [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.MethodInfo method, BindingFlags flags) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.PropertyInfo property) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.IsDefined (System.Reflection.PropertyInfo element, System.Type attributeType, Boolean inherit) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.HasAttribute (System.Reflection.MemberInfo member, System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Heuristics.StandardInjectionHeuristic.ShouldInject (System.Reflection.MemberInfo member) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector+<>c__DisplayClass3.<SelectPropertiesForInjection>b__2 (IInjectionHeuristic h) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.Any[IInjectionHeuristic] (IEnumerable`1 source, System.Func`2 predicate) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.<SelectPropertiesForInjection>b__1 (System.Reflection.PropertyInfo p) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateWhereIterator>c__Iterator1E`1[System.Reflection.PropertyInfo].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddRange (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.SelectPropertiesForInjection (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Strategies.PropertyReflectionStrategy.Execute (IPlan plan) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner+<>c__DisplayClass1.<CreateNewPlan>b__0 (IPlanningStrategy s) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForIEnumerableOfT.Map[IPlanningStrategy] (IEnumerable`1 series, System.Action`1 action) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.CreateNewPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.GetPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Providers.StandardProvider.Create (IContext context) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.ResolveInternal (System.Object scope) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.Resolve () [0x00000] in <filename unknown>:0 
  at Ninject.KernelBase+<>c__DisplayClass15.<Resolve>b__f (IBinding binding) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateSelectIterator>c__Iterator10`2[Ninject.Planning.Bindings.IBinding,System.Object].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateCastIterator>c__Iterator0`1[Nancy.Bootstrapper.IRegistrations].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations]..ctor (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToList[IRegistrations] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperWithRequestContainerBase`1[Ninject.IKernel].RegisterRegistrationTasks (IEnumerable`1 registrationTasks) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperBase`1[Ninject.IKernel].Initialise () [0x00000] in <filename unknown>:0 
  at Nancy.Owin.NancyMiddleware.UseNancy (Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, System.Action`1 configuration) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Startup.Configuration (IAppBuilder app) [0x00000] in <filename unknown>:0  System.NullReferenceException: Object reference not set to an instance of an object
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.get_ParentDefinitionMethodInfo () [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.MethodInfo method, BindingFlags flags) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.PropertyInfo property) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.IsDefined (System.Reflection.PropertyInfo element, System.Type attributeType, Boolean inherit) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.HasAttribute (System.Reflection.MemberInfo member, System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Heuristics.StandardInjectionHeuristic.ShouldInject (System.Reflection.MemberInfo member) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector+<>c__DisplayClass3.<SelectPropertiesForInjection>b__2 (IInjectionHeuristic h) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.Any[IInjectionHeuristic] (IEnumerable`1 source, System.Func`2 predicate) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.<SelectPropertiesForInjection>b__1 (System.Reflection.PropertyInfo p) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateWhereIterator>c__Iterator1E`1[System.Reflection.PropertyInfo].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddRange (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.SelectPropertiesForInjection (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Strategies.PropertyReflectionStrategy.Execute (IPlan plan) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner+<>c__DisplayClass1.<CreateNewPlan>b__0 (IPlanningStrategy s) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForIEnumerableOfT.Map[IPlanningStrategy] (IEnumerable`1 series, System.Action`1 action) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.CreateNewPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.GetPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Providers.StandardProvider.Create (IContext context) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.ResolveInternal (System.Object scope) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.Resolve () [0x00000] in <filename unknown>:0 
  at Ninject.KernelBase+<>c__DisplayClass15.<Resolve>b__f (IBinding binding) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateSelectIterator>c__Iterator10`2[Ninject.Planning.Bindings.IBinding,System.Object].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateCastIterator>c__Iterator0`1[Nancy.Bootstrapper.IRegistrations].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations]..ctor (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToList[IRegistrations] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperWithRequestContainerBase`1[Ninject.IKernel].RegisterRegistrationTasks (IEnumerable`1 registrationTasks) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperBase`1[Ninject.IKernel].Initialise () [0x00000] in <filename unknown>:0 
  at Nancy.Owin.NancyMiddleware.UseNancy (Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, System.Action`1 configuration) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Startup.Configuration (IAppBuilder app) [0x00000] in <filename unknown>:0 
2016/09/01 12:21:10.633 PlexRequests.UI.Program Fatal: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.get_ParentDefinitionMethodInfo () [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.MethodInfo method, BindingFlags flags) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.PropertyInfo property) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.IsDefined (System.Reflection.PropertyInfo element, System.Type attributeType, Boolean inherit) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.HasAttribute (System.Reflection.MemberInfo member, System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Heuristics.StandardInjectionHeuristic.ShouldInject (System.Reflection.MemberInfo member) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector+<>c__DisplayClass3.<SelectPropertiesForInjection>b__2 (IInjectionHeuristic h) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.Any[IInjectionHeuristic] (IEnumerable`1 source, System.Func`2 predicate) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.<SelectPropertiesForInjection>b__1 (System.Reflection.PropertyInfo p) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateWhereIterator>c__Iterator1E`1[System.Reflection.PropertyInfo].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddRange (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.SelectPropertiesForInjection (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Strategies.PropertyReflectionStrategy.Execute (IPlan plan) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner+<>c__DisplayClass1.<CreateNewPlan>b__0 (IPlanningStrategy s) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForIEnumerableOfT.Map[IPlanningStrategy] (IEnumerable`1 series, System.Action`1 action) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.CreateNewPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.GetPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Providers.StandardProvider.Create (IContext context) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.ResolveInternal (System.Object scope) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.Resolve () [0x00000] in <filename unknown>:0 
  at Ninject.KernelBase+<>c__DisplayClass15.<Resolve>b__f (IBinding binding) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateSelectIterator>c__Iterator10`2[Ninject.Planning.Bindings.IBinding,System.Object].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateCastIterator>c__Iterator0`1[Nancy.Bootstrapper.IRegistrations].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations]..ctor (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToList[IRegistrations] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperWithRequestContainerBase`1[Ninject.IKernel].RegisterRegistrationTasks (IEnumerable`1 registrationTasks) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperBase`1[Ninject.IKernel].Initialise () [0x00000] in <filename unknown>:0 
  at Nancy.Owin.NancyMiddleware.UseNancy (Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, System.Action`1 configuration) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Startup.Configuration (IAppBuilder app) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at Owin.Loader.DefaultLoader+<>c__DisplayClass12.<MakeDelegate>b__b (IAppBuilder builder) [0x00000] in <filename unknown>:0 
  at Owin.Loader.DefaultLoader+<>c__DisplayClass1.<LoadImplementation>b__0 (IAppBuilder builder) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.ResolveApp (Microsoft.Owin.Hosting.Engine.StartContext context) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.Start (Microsoft.Owin.Hosting.Engine.StartContext context) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.DirectHostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.HostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.StartImplementation (IServiceProvider services, Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start[Startup] (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0  System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.get_ParentDefinitionMethodInfo () [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.MethodInfo method, BindingFlags flags) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.GetParentDefinition (System.Reflection.PropertyInfo property) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.IsDefined (System.Reflection.PropertyInfo element, System.Type attributeType, Boolean inherit) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForMemberInfo.HasAttribute (System.Reflection.MemberInfo member, System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Heuristics.StandardInjectionHeuristic.ShouldInject (System.Reflection.MemberInfo member) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector+<>c__DisplayClass3.<SelectPropertiesForInjection>b__2 (IInjectionHeuristic h) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.Any[IInjectionHeuristic] (IEnumerable`1 source, System.Func`2 predicate) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.<SelectPropertiesForInjection>b__1 (System.Reflection.PropertyInfo p) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateWhereIterator>c__Iterator1E`1[System.Reflection.PropertyInfo].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[System.Reflection.PropertyInfo].AddRange (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at Ninject.Selection.Selector.SelectPropertiesForInjection (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Strategies.PropertyReflectionStrategy.Execute (IPlan plan) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner+<>c__DisplayClass1.<CreateNewPlan>b__0 (IPlanningStrategy s) [0x00000] in <filename unknown>:0 
  at Ninject.Infrastructure.Language.ExtensionsForIEnumerableOfT.Map[IPlanningStrategy] (IEnumerable`1 series, System.Action`1 action) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.CreateNewPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Planning.Planner.GetPlan (System.Type type) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Providers.StandardProvider.Create (IContext context) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.ResolveInternal (System.Object scope) [0x00000] in <filename unknown>:0 
  at Ninject.Activation.Context.Resolve () [0x00000] in <filename unknown>:0 
  at Ninject.KernelBase+<>c__DisplayClass15.<Resolve>b__f (IBinding binding) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateSelectIterator>c__Iterator10`2[Ninject.Planning.Bindings.IBinding,System.Object].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable+<CreateCastIterator>c__Iterator0`1[Nancy.Bootstrapper.IRegistrations].MoveNext () [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations].AddEnumerable (IEnumerable`1 enumerable) [0x00000] in <filename unknown>:0 
  at System.Collections.Generic.List`1[Nancy.Bootstrapper.IRegistrations]..ctor (IEnumerable`1 collection) [0x00000] in <filename unknown>:0 
  at System.Linq.Enumerable.ToList[IRegistrations] (IEnumerable`1 source) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperWithRequestContainerBase`1[Ninject.IKernel].RegisterRegistrationTasks (IEnumerable`1 registrationTasks) [0x00000] in <filename unknown>:0 
  at Nancy.Bootstrapper.NancyBootstrapperBase`1[Ninject.IKernel].Initialise () [0x00000] in <filename unknown>:0 
  at Nancy.Owin.NancyMiddleware.UseNancy (Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, Nancy.Owin.NancyOptions options) [0x00000] in <filename unknown>:0 
  at Owin.AppBuilderExtensions.UseNancy (IAppBuilder builder, System.Action`1 configuration) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Startup.Configuration (IAppBuilder app) [0x00000] in <filename unknown>:0 
  --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in <filename unknown>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <filename unknown>:0 
  at Owin.Loader.DefaultLoader+<>c__DisplayClass12.<MakeDelegate>b__b (IAppBuilder builder) [0x00000] in <filename unknown>:0 
  at Owin.Loader.DefaultLoader+<>c__DisplayClass1.<LoadImplementation>b__0 (IAppBuilder builder) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.ResolveApp (Microsoft.Owin.Hosting.Engine.StartContext context) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Engine.HostingEngine.Start (Microsoft.Owin.Hosting.Engine.StartContext context) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.DirectHostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.Starter.HostingStarter.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.StartImplementation (IServiceProvider services, Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at Microsoft.Owin.Hosting.WebApp.Start[Startup] (Microsoft.Owin.Hosting.StartOptions options) [0x00000] in <filename unknown>:0 
  at PlexRequests.UI.Program.Main (System.String[] args) [0x00000] in <filename unknown>:0 
```
#### Reproduction Steps:

Updating  from any version below 1.9 and clean installing gives me this error on start up.

```
Starting Up! Please wait, this can usually take a few seconds.
Version: 1.9.1
Location of the database: /opt/PlexRequest/PlexRequests.sqlite
Error loading type Typespec 0x1b000036 from  due to Could not resolve typespec token 1b000036

```

And then crashes with the log above.  

Versions below 1.9 start and function without any problems. For now I am stuck with 1.8.4 , which is working just fine.

```
Starting Up! Please wait, this can usually take a few seconds.
Version: 1.8.4
Location of the database: /opt/PlexRequest/PlexRequests.sqlite
Plex Requests is running on the following: http://+:3000/
All setup, Plex Requests is now ready!
```

**Comment by tidusjar** [comment]
Take a read of this:

https://plaza.quickbox.io/t/plex-request-net-dont-work/912/20

**Comment by AmansRevenger** [comment]
SInce I am on a RaspberryPi, those sources dont deploy newer versions of mono for me since mono-devel isnt even available in the first place.

Guess I am stuck for now?

**Comment by tidusjar** [comment]
Hmmm... That does suck... I'm not sure what else to suggest.

**Comment by AmansRevenger** [comment]
I will try moving it to an old netbook and forward that port to my website. Also it works for my needs with 1.8.4 , missing out on features is "not optimal" but not exactly deal breaking.

Thanks anyway!

**Comment by tidusjar** [comment]
Yeah, the codebase has changed a lot between 1.8.4 - 1.9 

Needed to change a few things to make sure the application preforms fast and extensible. 

I assumed that It would work for anyone with mono, but obviously that is not the case. 

Sorry about that.

**Comment by deathbybandaid** [comment]
I know this was closed, but has anybody found anything to fix this? I'm on rasbian

I spent hours on the web trying to get mono to be a version past 3.2. I was about to give up, but I unistalled mono, and reinstalled it, and was pleased when I ran mono --version

Mono JIT compiler version 5.3.0 (master/4b9235f Sat  8 Apr 21:02:26 EDT 2017)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       normal
        Notifications: epoll
        Architecture:  armel,vfp+hard
        Disabled:      none
        Misc:          softdebug
        LLVM:          supported, not enabled.
        GC:            sgen (concurrent by default)


So now, I am running a mono version "v4.x or above",,, but I still get the error 

Error loading type Typespec 0x1b000036 from  due to Could not resolve typespec token 1b000036
